### ISSUE TITLE
Using tsconfig `"references"` option without `"include"` in the default tsconfig means extension finds no files to parse

### ISSUE THREAD

**Issue by ethan-fraser** [iulian-radu-at/find-unused-exports#33]
**Describe the bug**
If you use the `"references"` tsconfig option to specify multiple other tsconfigs from the default one, without the `"include"` option in the default one, then Find Unused Exports will not read any files.

With a `.findUnusedExports.json` in the root of the project, I can get it to read the files. But I am the only dev who uses this extension on my team, so don't want to commit this config file. It appears that I have a `.vscode/find-unused-exports.json` file - but this does not appear to have any effect.

**Expected behavior**
Either traverse the tsconfig references to find all sources from all available tsconfigs; or make the `.vscode/find-unused-exports.json` file work.

**To Reproduce**
Have a `tsconfig.json` in the root of the project with no `"include"` option. For example, mine looks like this:
```json
{
  "files": [],
  "references": [
    { "path": "./tsconfig.app.json" },
    { "path": "./tsconfig.node.json" }
  ],
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@/*": ["./src/*"]
    }
  }
}
```
A config set up like this can be easily set up using `npm create vite@latest`.

**Information about the extension:**
 - Version: v2.1.5
 - Enabled features
   - [x] find unused exports
   - [x] find circular dependecies

**Information about your project:**
 - Type of files:
   - [ ] .js / .jsx
   - [x] .ts / .tsx
 - [x] It has a tsconfig.json file 
 - Number of files
   - 326 processed by the extension

**Screenshots**

**Additional context**
If I remove the `"files"` option or add the `"include"` option to my tsconfig.json, this works to get Find Unused Exports to read my files, but it raises other errors in my tsconfig.

**Comment by ethan-fraser** [comment]
Sorry, just realised this has already been raised by #30. Feel free to close.

**Comment by iulianraduat** [comment]
Closed as duplicates #30
